mangaki/mangaki/algo/fma.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+import pywFM
+import os
+from scipy.sparse import coo_matrix
+
+from mangaki.algo.recommendation_algorithm import (RecommendationAlgorithm,
+                                                   register_algorithm)
+
+
+@register_algorithm('fma', {'rank': 20})
+class MangakiFMA(RecommendationAlgorithm):
+    def __init__(self, rank=20, nb_iterations=10):
+        super().__init__()
+        self.rank = rank
+        self.nb_iterations = nb_iterations
+        self.fm = None
+
+    def load(self, filename):
+        backup = super().load(filename)
+        self.mu = backup.mu
+        self.W = backup.W
+        self.V = backup.V
+        self.V2 = backup.V2
+
+    @property
+    def is_serializable(self):
+        return False  # Not yet
```

Comment:
Nit: if it is false, it's obvious this is not yet serializable :p.

Reply:
Actually, now it is.

Reply:
It was from CFM.